packages/ingest/src/granule.ts

```diff
@@ -199,8 +199,15 @@ export async function handleDuplicateFile(params: {
   target: { Bucket: string, Key: string },
   duplicateHandling: DuplicateHandling,
   checksumFunction?: (bucket: string, key: string) => Promise<[string, string]>,
-  syncFileFunction?: (bucket: string, key: string) => Promise<void>,
-  ACL?: string
+  syncFileFunction?: (params: {
```

Comment:
Updates syncFileFunction to allow for the optional bucket parameter.  Adds optional sourceBucket key to params.